herbert_core/utilities/maths/replicate_logarray.m

```diff
@@ -1,26 +1,44 @@
 function vout = replicate_logarray (v, n)
 % Replicate logical array elements according to list of repeat indicies
 %
-%   >> ivout = replicate_logarray (v, n)
+%   >> vout = replicate_logarray (v, n)
```

Comment:
given this is legacy back comptaibility, I guess we'll leave the name, but log could mean logarithm